configs/sites/tier1/jet/mirrors.yaml

```diff
@@ -1,15 +1,15 @@
 mirrors:
   local-source:
     fetch:
-      url: file:///mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/source-cache
+      url: file:///contrib/spack-stack/mirror
       access_pair:
       - null
       - null
       access_token: null
       profile: null
       endpoint_url: null
     push:
-      url: file:///mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/source-cache
+      url: file:///contrib/spack-stack/mirror
```

Comment:
My understanding is that space on /contrib is a bit more precious than lfs. Not that we're not allowed to put caches there, but from a being-good-neighbors perspective we might want to consider leaving the caches on lfs, and just have the installations themselves under /contrib.